### ISSUE TITLE
'running dataset construction code ' isssue

### ISSUE THREAD

**Issue by xz0613** [HansDahleKvadsheim/Sea-Pix-GAN-Group-6#1]
input_data_dir = "./UnderwaterImages/input"
gt_data_dir = "./UnderwaterImages/ground_truth";
Do I have to prepare the underwater images myself?

**Comment by xz0613** [comment]
Are you splitting the paired dataset trainA, trainB, validation in the EUVP dataset into two folders UnderwaterImages/input, UnderwaterImages/ground_truth?

**Comment by HansDahleKvadsheim** [comment]
Yes, due to the file size, you'll have to prepare the images yourself. We added to link to the dataset in the readme. There you'll find a folder with underwater-images along with their respective ground truths. Hope this helps.

**Comment by xz0613** [comment]
I'd like to ask what python and pytorch versions and conda environment your used during your experiments

**Comment by HansDahleKvadsheim** [comment]
We ran the experiments in a Kaggle environment, not Conda, to access their remote GPUs. This enviroment used Pytorch 2.1.2 and Python 3.10.13
